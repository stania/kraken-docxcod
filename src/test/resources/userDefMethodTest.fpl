disk-usage-summary

<#list .vars["disk-usage-summary"] as u>
===========
Host: ${u["host"]}
-----------${makeNewChart("ctx", u)}
<#list u["disk-usage"] as d>
${d["disk-name"]}: ${d["usage"]}%
</#list>
</#list>
===========
