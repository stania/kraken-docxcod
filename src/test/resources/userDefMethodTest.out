disk-usage-summary

===========
Host: SAN Management
-----------
C:\: 10%
D:\: 0%
E:\: 30%
===========
Host: GMS Server 02
-----------
===========
Host: IIS_1
-----------
C:\: 20%
D:\: 0%
===========
